src/Analyzer.Reports/ConsoleReportWriter.cs

```diff
@@ -15,6 +15,17 @@ public class ConsoleReportWriter : IReportWriter
         internal static string IndentedNewLine = Environment.NewLine + "\t";
         internal static string TwiceIndentedNewLine = Environment.NewLine + "\t\t";
 
+        private IFileInfo configurationsFile;
+
+        /// <summary>
+        /// Constructor of the SarifReportWriter class
+        /// </summary>
+        /// <param name="configurationsFilePath">The configuration file that will be used</param>
+        public ConsoleReportWriter(IFileInfo configurationsFilePath = null)
```

Comment:
There might not be a need to store the configuration file path.

Reply:
I believe you mentioned adding it to the Sarif or something?